doc/security/_index.md

```diff
@@ -0,0 +1,111 @@
+---
+title: "OpenTitan Security Introduction"
+---
+
+## Overview
+
+OpenTitan’s mission is to create a trustworthy, vendor-agnostic open source silicon Root of Trust (RoT) widely adopted across the industry.
+The project aims to accomplish this by implementing strong logical security integrity guarantees in the RTL and firmware components, and by providing a comprehensive certification plan for physical implementations.
+
+## Logical Security Model
+
+The [OpenTitan Security Model][2] provides a high level framework for device provisioning and run-time operations.
+It starts by enumerating the range of logical entities supported by the architecture, and their mapping into software stages.
+Runtime isolation properties and baseline identity concepts are introduced in this document.
+
+## Functional Guarantees
+
+At the functional level OpenTitan aims to provide the following guarantees:
+
+* Silicon Owners shall be able to deploy their own Root of Trust (RoT) Public Key Infrastructure (PKI) after taking ownership of the device.
+* Silicon Creators shall endorse the authenticity of the hardware.
+  Endorsement is contingent on the silicon adhering to the physical implementation guidelines and certification requirements stipulated by the project.
+  The endorsement shall be measurable via a Transport Certificate signed by the Silicon Creator and lowRISC.
+* OpenTitan shall provide full boot attestation measurements to allow Silicon Owners to verify the boot chain configuration.
+  The attestation chain shall be anchored in the Silicon Owner’s RoT PKI.
+* OpenTitan shall provide a key manager implementation strongly binded to the boot chain.
```

Comment:
binded -> bound sounds a bit more natural to me.

Reply:
Done